nucypher/crypto/powers.py

```diff
@@ -460,6 +465,8 @@ def _get_static_secret_from_ritual_id(self, ritual_id: int) -> SessionStaticSecr
     def get_pubkey_from_ritual_id(self, ritual_id: int) -> SessionStaticKey:
         return self._get_static_secret_from_ritual_id(ritual_id).public_key()
 
+
+class ThresholdRequestDecryptingPower(ThresholdRequestPower):
```

Comment:
Let's rename to match format of Signing power?

```suggestion
class ThresholdDecryptionRequestPower(ThresholdRequestPower):
```

Reply:
✔️